troubleshooting/debugging.md

```diff
@@ -479,3 +480,30 @@ Path                /logs/request.log
 ```
 
 In other cases, the original config would be correct. For example, assume your app produces first a log file `request.log` and then next creates and writes new logs to a new file named `request.log1` and then next `request.log2`. And then when the max number of files are reached, the original `request.log` is deleted and then a new `request.log` is written. If that is the scenario, then the original Path pattern with a wildcard at the end is correct since it is necessary to match all log files where new logs are written. This method of log rotation is the behavior of the [log4j DirectWriteRolloverStrategy](https://logging.apache.org/log4j/2.x/manual/appenders.html#DirectWriteRolloverStrategy) where there is no file renaming. 
+
+
+#### FireLens Tag and Match Pattern and generated config
+
+If you use Amazon ECS FireLens to deploy Fluent Bit, then please be aware that some of the Fluent Bit configuration is managed by AWS. ECS will add a log input for your stdout/stderr logs. This single input captures all of the logs from each container that uses the `awsfirelens` log driver. 
+
+See this repo for [an example of a FireLens generated config](https://github.com/aws-samples/amazon-ecs-firelens-under-the-hood/blob/mainline/generated-configs/fluent-bit/generated_by_firelens.conf). See the [FireLens Under the Hood](https://aws.amazon.com/blogs/containers/under-the-hood-firelens-for-amazon-ecs-tasks/) blog for more on how FireLens works.
+
+In Fluent Bit, each log ingested is given a [Tag](https://docs.fluentbit.io/manual/concepts/key-concepts). FireLens assigns tags to stdout/stderr logs from each container in your task with the following pattern:
+
+```
+{container name in task definition}-firelens-{task ID}
+```
+
+So for example, if your container name is `web-app` and then your Task ID is `4444-4444-4444-4444` then the tag assigned to stdout/stderr logs from the `web-app` container will be:
+
+```
+web-app-firelens-4444-4444-4444-4444
+```
+
+If you use a custom config file with FireLens, then you would want to set a `Match` pattern for these logs like:
+
+```
+Match web-app-firelens*
```

Comment:
Hi again! I was just looking through your repo for an example of this exact thing, and your PR here came up in my search! Very fun timing.

My specific use case here is wanting to match on everything coming out of firelens, so I can send it to an error monitoring service. I imagine this more common use case than wanting to match on a specific container. So my suggestion is that this section have something that reflects that, like so:

```
Match *firelens*
```

```
[FILTER]
    Name rewrite_tag
    Match *firelens*
    Rule $level ^(error|ERROR) errors true
```

Just my 2 cents - I've since figured out my own configuration thanks to all your other docs. Thanks so much!

Reply:
Good idea, I'll add that :)

Reply:
That helps makes it clear that Match is not just for prefix matching